CHANGELOG.md

```diff
@@ -26,6 +26,7 @@ This project uses the changelog in accordance with [keepchangelog](http://keepac
  - Added support for timestamped comments in CLI via `rem`, `;`, `%` or `#` (@doegox)
  - Fixed watchdog trigger during `hw factory_reset` (@doegox)
  - Added PyInstaller support for CLI client (@augustozanellato)
+ - Parallelize mfkey32v2 processes called from CLI (@p-l-)
```

Comment:
new changelog entries should be moved to the top of the list

Reply:
Sure, makes sense, sorry. That's done!